tljh-voila-gallery/tljh_voila_gallery/__init__.py

```diff
@@ -34,9 +34,11 @@ def options_form(spawner):
         return jinja2.Template(f.read()).render(examples=gallery['examples'])
 
 class GallerySpawner(DockerSpawner):
-    # FIXME: What to do about idle culling?!
     cmd = 'jupyter-notebook'
 
+    # rm containers when they stop
+    remove = True
```

Comment:
This is not picked up when the server is shut down (still shows in `docker ps -a`).

Probably because it is used only when its `stop` method is called, not when the notebook server (pid 1) stops.